ingest/mappings/dataset.json

```diff
@@ -0,0 +1,208 @@
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "dcoCommunities": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "portalGroups": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "projects": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "citations": {
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "uri": {
+            "type": "string"
+          }
+        }
+      },
+      "distributions": {
+        "properties": {
+          "accessURL": {
+            "type": "string"
+          },
+          "files": {
+            "properties": {
+              "downloadURL": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              },
+              "uri": {
+                "type": "string"
+              }
+            }
+          },
+          "name": {
+            "type": "string"
+          },
+          "uri": {
+            "type": "string"
+          }
+        }
+      },
+      "dcoId": {
+        "type": "string"
+      },
+      "doi": {
+        "type": "string"
+      },
+      "isDcoPublication": {
+        "type": "boolean"
+      },
+      "mostSpecificType": {
```

Comment:
We should support a not indexed version of mostSpecificType so it can be used as a facet value.

```
"mostSpecificType": {
        "type":"string",
        "index": "not_analyzed"
      },
```